mmv1/products/privateca/terraform.yaml

```diff
@@ -144,6 +144,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       method_name_separator: ':'
       parent_resource_attribute: ca_pool
       example_config_body: 'templates/terraform/iam/example_config_body/privateca_ca_pool.tf.erb'
+      iam_conditions_request_type: :REQUEST_BODY
```

Comment:
What is the different between value `REQUEST_BODY` and `QUERY_PARAM`? Is `REQUEST_BODY` the correct value here?

Reply:
based on discussion elsewhere, I think so?